src/jvm.ts

```diff
@@ -427,6 +427,11 @@ class JVM {
         assert(false, `Invariant failure: Thread pool cannot be emptied post-JVM termination.`);
         return false;
       case JVMStatus.TERMINATING:
+
+        if (!RELEASE) {
```

Comment:
You should add a command line flag for dumping these stats (e.g. `-XdumpJitStats` or something).